app/models/user.rb

```diff
@@ -19,6 +19,14 @@ def display_name
     username.presence || full_name
   end
 
+  def initials
+    if full_name.blank?
+      first_name.slice(0, 2)
+    else
+      full_name.split(' ', 2).map(&:chr).join
```

Comment:
I would pick the first letter from the first and **last** name, instead of the two first names